### DISCUSSION TITLE
Running without docker

### DISCUSSION THREAD

**Discussion by tecosaur** [jeffvli/feishin#846] [Q&A]
Hello,

For convenience, I'm thinking of selfhosting this myself. Ideally, I'd like to do this with a bare-metal setup. From a glance at the Dockerfile it looks at though this may be as simple as:

- Running `npm run build:web`
- Statically serving `app/release/app/dist/web`

I feel like I'm probably missing one or two things though, so advice would be much appreciated!

**Comment by kgarner7** [comment]
That's what I personally do. The docker container is just NGINX serving static files, so you can adapt the same rules (or use any other reverse proxy)

**Reply by tecosaur** [reply]
Thanks for chiming in, that's encouraging to hear!

I guess the one thing that's not clear to me is how the env vars work if it's just static content?

**Reply by kgarner7** [reply]
Check `settings.js.template`. In the Docker image, this file is served statically based off of environment variables, but you can just put this in the same place
